### PR TITLE
update link to new ia nav blog post

### PR DESCRIPTION
**Pull request by jacobheric**
## Description

Blog link changed.

## Motivation
Correct link

## Type of change (choose one)
- [x] Chore (refactors, upgrades, etc.)
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Security fix (non-breaking change that fixes a potential vulnerability)
- [ ] Docs
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist
- [x] I've linked any associated issues to this PR.
- [x] I've tested my own changes.

*[Check our Pull Request Guidelines](https://github.com/inngest/inngest/blob/main/docs/PULL_REQUEST_GUIDELINES.md)*

### CODE REVIEW

File: ui/apps/dashboard/src/components/Navigation/Alert.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ export const Alert = () => {
             onClick={() => dismiss()}
           />
         </div>
-        <Link href="http://www.inngest.com/blog/reimaging-our-ia" className="mt-4">
+        <Link href="http://inngest.com/blog/inngest-new-design" className="mt-4">
```
Comment (djfarrelly) [inline ui/apps/dashboard/src/components/Navigation/Alert.tsx:37]:
```suggestion
        <Link href="https://www.inngest.com/blog/inngest-new-design" className="mt-4">
```

The site is on www subdomain now and just added https :)

**Review by djfarrelly — APPROVED** [review]
Approved - but with suggestion

File: ui/apps/dashboard/src/components/Navigation/Alert.tsx (hunk 1 above)
Comment (jacobheric) [reply ui/apps/dashboard/src/components/Navigation/Alert.tsx:37]: thanks!

**Comment by jacobheric** [comment]
I got a new url from Charly this AM. Updated.

**Pull request merged**